README.md

```diff
@@ -17,7 +17,7 @@ Supported operating systems:
 
 DISCLAIMER: The driver is provided as is, without warranty.
 
-Full documentation can be found in the IBM knowlage center -> HERE
+Full documentation can be found in the [IBM knowlage center](https://www.ibm.com/support/knowledgecenter/SSRQ8T).
```

Comment:
The link is broken, i believe it is because we haven't created the page yet, right?

Reply:
correct. it will only go live at GA.